### PR TITLE
Better description?

### PR DESCRIPTION
**Pull request by tisnik**
# Description

Better description?

## Type of change

- Documentation update

## Testing steps

N/A

## Checklist
* [ ] `make before_commit` passes
* [ ] updated documentation wherever necessary
* [ ] added or modified tests if necessary
* [ ] updated schemas and validators in [insights-data-schemas](https://github.com/RedHatInsights/insights-data-schemas) in case of input/output change

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,11 +12,14 @@ Aggregator service for insights results
 
 ## Description
 
-Insights Results Aggregator is a service that provides Insight OCP data that are being consumed by
-OpenShift Cluster Manager. That data contain information about clusters status (especially health,
-security, performance) based on results generated by Insights rules engine. Insights OCP data are
-consumed from selected broker, stored in a storage (that basically works as a cache) and exposed via
-REST API endpoints.
+Insights Results Aggregator is a service that provides Insight OCP data
+(offically called *recommendations*) that are being consumed by OpenShift
```
Comment (Bee-lee) [inline README.md:16]: typo:)

Comment (joselsegura) [reply README.md:16]: But wasn't the new kid in the block "remediations"?

Comment (tisnik) [reply README.md]: fixed :)

Comment (tisnik) [reply README.md]: @joselsegura not now, probably next week perhaps :)

**Review by juandspy — APPROVED** [review]

**Review by Bee-lee — APPROVED** [review]

**Pull request merged**